fenix/app/src/main/java/org/mozilla/fenix/components/appstate/AppState.kt

```diff
@@ -79,4 +81,5 @@ data class AppState(
     val wallpaperState: WallpaperState = WallpaperState.default,
     val standardSnackbarError: StandardSnackbarError? = null,
     val shoppingState: ShoppingState = ShoppingState(),
+    val awesomeBarState: AwesomeBarState = AwesomeBarState(),
```

Comment:
I think we talked about using a-c's `BrowserStore` to hold this state, instead of Fenix's `AppStore`—do folks have opinions on that?

Reply:
No, having the here is fine and is in line with [our original reasoning](https://github.com/mozilla-mobile/fenix/issues/20704#issuecomment-956522471) for having an `AppStore`.